app/utils/notification/index.ts

```diff
@@ -90,12 +90,26 @@ export const emitNotificationError = (type: 'Team' | 'Channel' | 'Post' | 'Conne
 
 export const scheduleExpiredNotification = (serverUrl: string, session: Session, serverName: string, locale = DEFAULT_LOCALE) => {
     const expiresAt = session?.expires_at || 0;
-    const expiresInDays = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asDays()));
+    const expiresInHours = Math.ceil(Math.abs(moment.duration(moment().diff(moment(expiresAt))).asHours()));
+    const expiresInDays = Math.floor(expiresInHours / 24); // Calculate expiresInDays
+    const remainingHours = expiresInHours % 24; // Calculate remaining hours
     const intl = createIntl({locale, messages: getTranslations(locale)});
+    
+    const daysPlural = intl.formatPlural(expiresInDays, {
+        one: 'day',
+        other: 'days',
+    });
+
+    const hoursPlural = intl.formatPlural(remainingHours, {
+        one: 'hour',
+        other: 'hours',
+    });
+
     const body = intl.formatMessage({
         id: 'mobile.session_expired',
-        defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire every {daysCount, number} {daysCount, plural, one {day} other {days}}.',
-    }, {siteName: serverName, daysCount: expiresInDays});
+        defaultMessage: 'Please log in to continue receiving notifications. Sessions for {siteName} are configured to expire in {daysCount, number} {days} and {hoursCount, number} {hours}.', // Update the message
```

Comment:
@esethna can you please advice on the message that we should use for hours and days ?

Reply:
We probably also need to remove the whole `days` part if the number of days is 0.